icons/noaa.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>NOAA - National Oceanic and Atmospheric Administration</title></svg><path d="M12.549.012A12 12 0 0 0 2.22 5.048c2.494 1.323 4.242 5.228 6.584 7.372 2.341 2.18 4.576 2.893 6.901-.37 1.768-2.241 2.757-6.762 5.838-7.327A12 12 0 0 0 12.55.013Zm-1.432 4.653a.613.613 0 0 1 .626.6v1.946a.726.726 0 0 1-.63.584h-.771a.71.71 0 0 1-.633-.574V5.275a.626.626 0 0 1 .633-.6h.772zm-3.921.01h1.297a.668.668 0 0 1 .73.606v2.514h-.63l.004-2.287c0-.142-.068-.227-.191-.227h-.578v2.514h-.632Zm5.675 0h.782a.606.606 0 0 1 .633.58v2.54h-.633V6.621h-.782v1.174h-.629V5.281a.671.671 0 0 1 .63-.606zm2.543 0h.753a.606.606 0 0 1 .629.58v2.54h-.63V6.621h-.768v1.174h-.633V5.281a.675.675 0 0 1 .649-.606zm-4.868.59a.204.204 0 0 0-.204.205v1.543a.201.201 0 0 0 .207.201h.36a.204.204 0 0 0 .205-.204V5.463a.204.204 0 0 0-.208-.198Zm2.562.033a.234.234 0 0 0-.214.227v.512h.775l-.016-.512a.224.224 0 0 0-.22-.227Zm2.514 0a.23.23 0 0 0-.208.227v.512h.753v-.512a.224.224 0 0 0-.22-.227zm7.141 1.397a12 12 0 0 1 .004.01l.006-.01zm.004.01c-1.71 2.571-4.538 6.154-7.84 7.35l-.408.171a.928.928 0 0 0 .25.075h.39c.836-.032 2.043-.386 2.289.464a9.908 9.908 0 0 0-4.784 1.271c-1.495.82-3.39 1.56-4.628.409.992-.133 2.825-.733 3.47-1.32-5.247-.072-7.58-3.341-9.22-5.128-.59-.684-1.197-1.258-1.817-1.316a12 12 0 1 0 22.298-1.976Z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>NOAA</title><path d="M12.549.012A12 12 0 0 0 2.22 5.048c2.494 1.323 4.242 5.228 6.584 7.372 2.341 2.18 4.576 2.893 6.901-.37 1.768-2.241 2.757-6.762 5.838-7.327A12 12 0 0 0 12.55.013Zm-1.432 4.653a.613.613 0 0 1 .626.6v1.946a.726.726 0 0 1-.63.584h-.771a.71.71 0 0 1-.633-.574V5.275a.626.626 0 0 1 .633-.6h.772zm-3.921.01h1.297a.668.668 0 0 1 .73.606v2.514h-.63l.004-2.287c0-.142-.068-.227-.191-.227h-.578v2.514h-.632Zm5.675 0h.782a.606.606 0 0 1 .633.58v2.54h-.633V6.621h-.782v1.174h-.629V5.281a.671.671 0 0 1 .63-.606zm2.543 0h.753a.606.606 0 0 1 .629.58v2.54h-.63V6.621h-.768v1.174h-.633V5.281a.675.675 0 0 1 .649-.606zm-4.868.59a.204.204 0 0 0-.204.205v1.543a.201.201 0 0 0 .207.201h.36a.204.204 0 0 0 .205-.204V5.463a.204.204 0 0 0-.208-.198Zm2.562.033a.234.234 0 0 0-.214.227v.512h.775l-.016-.512a.224.224 0 0 0-.22-.227Zm2.514 0a.23.23 0 0 0-.208.227v.512h.753v-.512a.224.224 0 0 0-.22-.227zm7.141 1.397a12 12 0 0 1 .004.01l.006-.01zm.004.01c-1.71 2.571-4.538 6.154-7.84 7.35l-.408.171a.928.928 0 0 0 .25.075h.39c.836-.032 2.043-.386 2.289.464a9.908 9.908 0 0 0-4.784 1.271c-1.495.82-3.39 1.56-4.628.409.992-.133 2.825-.733 3.47-1.32-5.247-.072-7.58-3.341-9.22-5.128-.59-.684-1.197-1.258-1.817-1.316a12 12 0 1 0 22.298-1.976Z"/></svg>
```

Reply:
Fixed the title, when I readjusted the icon size to exactly 24.0000px